crates/typst-library/src/text/raw.rs

```diff
@@ -56,6 +62,35 @@ use crate::World;
 /// #raw("fn " + "main() {}", lang: "rust")
 /// ```
 ///
+/// # Styling
+///
+/// By default, the `raw` element uses the `DejaVu Sans Mono` font, available
+/// by default in Typst, with a smaller font size of `0.8em` (that is, 80% of
+/// the global font size). You can customize these properties with a show-set
+/// rule:
+///
+/// ````example
+/// // Switch to Cascadia Code for both inline and block raw.
+/// #show raw: set text(font: "Cascadia Code")
+/// // Make raw blocks 20% larger than normal text,
+/// // while keeping inline raw smaller than text.
+/// #show raw.where(block: true): set text(1.2em)
```

Comment:
They aren't really 20% larger than normal text because it stacks with the 0.8em. So the end result is 96% of the size.

Monospace fonts are for some reason just way larger than normal fonts. [Browsers also have a weird quirks due to this.](https://meyerweb.com/eric/thoughts/2010/02/12/fixed-monospace-sizing/)